R/workflow.R

```diff
@@ -69,54 +69,56 @@ print.workflow <- function(x, ...) {
 
 print_header <- function(x) {
   if (x$run) {
-    fit <- " [fit]"
+    fit <- " [trained]"
   } else {
     fit <- ""
   }
 
-  cat_line(glue::glue("<workflow{fit}>"))
+  cat(cli::rule(glue::glue("model workflow {fit}")))
 
   invisible(x)
 }
 
 print_preprocessor <- function(x) {
   actions <- names(x$pre$actions)
-  preprocessor <- tab("<preprocessor>")
 
   if ("formula" %in% actions) {
-    type <- tab("Formula", 2L)
+    cat_line(cli::style_bold("\nformula\n"))
```

Comment:
If we do the capital `Workflow`, I think I'd rather be consistent and have a capital `Formula` and `Recipe` and `Model` as well like before